### PR TITLE
Update required for Cocoapod integration

### PR DESCRIPTION
**Pull request by macmagician**

### CODE REVIEW

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 1]
```diff
@@ -30,18 +30,39 @@ This SDK has a number of key benefits:
...
 - Xcode 11
 - Swift 5
+- Cocoapods
 
 ## Xcode Integration
+The VUDRMFairPlay SDK is distributed using Cocoapods. Projects set up with Cocoapods use an Xcode workspace, so it is important that projects with Cocoapods are always opened using the workspace file instead of the project file. Our simple demo applictation demonstrates the required set up for Cocoapods.
 
-1. Copy the vudrmFairPlay.framework into your application's project directory
-2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’:
+If you are integrating the VUDRMFairPlay SDK into your own project, please ensure that the project has been set up correctly for Cocoapods. There are numerous online turtorials demonstrating how to do this. You can then replace or merge the`podfile` with the one in our demo application.
```
Comment (fschurer) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md]: Typo in 'tutorials'

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 2]
```diff
@@ -30,18 +30,39 @@ This SDK has a number of key benefits:
...
+
+It is important that the latest version of Cocoapods is installed before the demo project or your project can be opened without error.
+
+1. You can ensure that you have the latest version of Cocoapods by opening a new shell in the Terminal application and entering:
+
+	`sudo gem install cocoapods`
+
+2. To pull vudrmFairPlay.framework in to the demo project or your project with Cocoapods, ensure that you quit Xcode if it is running, then open the Temrinal application and navigate to the project directory, for example:
```
Comment (fschurer) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md]: Typo in 'Terminal'

File: docs/source/integrations/mobile/ios-fairplay-sdk.md [hunk 3]
```diff
@@ -30,18 +30,39 @@ This SDK has a number of key benefits:
...
 - Xcode 11
 - Swift 5
+- Cocoapods
 
 ## Xcode Integration
+The VUDRMFairPlay SDK is distributed using Cocoapods. Projects set up with Cocoapods use an Xcode workspace, so it is important that projects with Cocoapods are always opened using the workspace file instead of the project file. Our simple demo applictation demonstrates the required set up for Cocoapods.
 
-1. Copy the vudrmFairPlay.framework into your application's project directory
-2. Navigate to your target and add vudrmFairPlay.framework to your ‘Embedded Binaries’. You should see that vudrmFairPlay.framework also now appears under ‘Linked Frameworks and Libraries’:
+If you are integrating the VUDRMFairPlay SDK into your own project, please ensure that the project has been set up correctly for Cocoapods. There are numerous online tutorials demonstrating how to do this. You can then replace or merge the`podfile` with the one in our demo application.
```
Comment (PerritonBranch) [inline docs/source/integrations/mobile/ios-fairplay-sdk.md]:
```suggestion
If you are integrating the VUDRMFairPlay SDK into your own project, please ensure that the project has been set up correctly for Cocoapods. There are numerous online tutorials demonstrating how to do this. You can then replace or merge the `podfile` with the one in our demo application.
```

File: docs/source/integrations/mobile/ios-fairplay-sdk.md (hunk 1 above)
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: Fixed, thank you.

File: docs/source/integrations/mobile/ios-fairplay-sdk.md (hunk 3 above)
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: Fixed, thank you.

File: docs/source/integrations/mobile/ios-fairplay-sdk.md (hunk 2 above)
Comment (macmagician) [reply docs/source/integrations/mobile/ios-fairplay-sdk.md]: Fixed, thank you.

**Pull request merged**